support-services/docker-compose.yml

```diff
@@ -140,3 +140,12 @@ services:
         condition: service_healthy
     ports:
       - '8081:8080'
+
+  tools-pipelines:
+    image: geonetwork/geonetwork-ui-tools-pipelines:latest
+    environment:
+      ES_HOST: http://elasticsearch:9200
+      RECORDS_INDEX: gn-records
+    depends_on:
+      init:
```

Comment:
Do you think you could invert the dependency, and make `init` depend on this? `init` will do a reindexation of the catalog so that would mean the md quality widget would show up right away without needing a manual indexation